### DISCUSSION TITLE
Feature Request: Location/Tag BI Reporting

### DISCUSSION THREAD

**Discussion by a2brew** [sysadminsmedia/homebox#729] [Ideas]
### What is the problem you are trying to solve with this feature?

Now that the new maintainer team has added 'cost by location' roll-up (thank you!!), I think another cool feature to add would be basic pie/bar chart reports, showing the total cost by location & tag.

### What is the solution you are proposing?

In the short-term, it would be valuable to have a separate 'reports' tab (similar to 'Profile', 'Search') which then showed some basic reporting about total cost/items by tag or location. Here are some initial *hopefully* simple ideas for v1 of this feature:

* Pie Chart: Cost by Location
* Pie Chart: Cost by Tag
* Pie Chart: Cost by Insured/Not Insured
* Bar Chart: Number of Items by Location
* Bar Chart: Number of Items by Tag

For later versions, I think it would be cool to see some of these same insights 'over time', based on the purchase/sale dates entered for each item. That way the user can track certain trends over time.

### What alternatives have you considered?

Once external database functionality is added, I suppose it will be possible to build out these reports via a separate open-source BI tool. However, I think this would be a really cool option to have 'natively' within Homebox. 

### Contributions

- [X] I have searched through existing issues and feature requests to see if my idea has already been proposed.
- [ ] If this feature is accepted, I would be willing to help implement and maintain this feature.
- [X] If this feature is accepted, I'm willing to sponsor the development of this feature.

**Comment by katosdev** [comment]
Potential issue with location pricing needs resolving first, had a report that quantity isn’t being acknowledged for location pricing.

**Comment by tankerkiller125** [comment]
Please note we're converting this issue into a discussion as per our new procedures/policies. You can read more about that in our announcement #712
